src/cli/templates/create/webpack.config.js

```diff
@@ -0,0 +1,159 @@
+const path = require('path')
+const nodeExternals = require('webpack-node-externals')
+const pkg = require('./package.json')
+
+const cleanVersion = semver => semver.replace(/[>=]/g, '')
```

Comment:
Not very reliable? What if they change the version to be caret-range.
`semver.clean` should to the trick? https://www.npmjs.com/package/semver

Reply:
Oh, I'm wrong, it's not doing what I expected. Not sure what's the best way to go here. Use `current` too and let the user fix it if they need something different?

Reply:
BTW why do we need babel config in package.json at all of it's explicitly set here?

Reply:
You are right. I noticed this yesterday too, when I locally changed `engines.node` to `>=4.6 <9` because `botpress` doesn't like `node@9`...

I was thinking that I should implement a separate utility, devDepended here, to:
 - take all the released versions of node via [node-version-data](https://www.npmjs.com/package/node-version-data),
 - get the expected version from `engines.node` or possibly using [expected-node-version](https://www.npmjs.com/package/expected-node-version) for speed and compatibility,
 - filter out only the ones which satisfy `engines.node` using `semver.satisfies()`,
 - then pick the lowest version.

Good idea?

Reply:
Yeah great plan. There's `minSatisfying` already in `semver` btw

Reply:
Perfect. Thanks for the tip.

Reply:
Regarding `babel` in `package.json`: I don't remember. I'll have to reconsider it (possibly clean it up) when I address the whole `cleanVersion` thing with the utility lib.

I remember there are at least two things going on, which need different configs: web and node. But that doesn't explain *three* configs :)

Reply:
Now using [minimum-node-version](https://www.npmjs.com/package/minimum-node-version) instead of `cleanVersion(pkg.engines.node)`.

Reply:
Re: `babel` in `package.json` again... I see now, and remember, that the `babel` config in `package.json` is `babel: {env: {test: {presets, plugins}}}`. So there are three distinct configs going on:

- web
- node
- node during test

They are all different, and needed :)